### PR TITLE
Issue #5 add Genesis Schaerrer

### LINKED ISSUE #5: Add Genesis Schaerrer
**Please add your name to the contribution-list.md**

### PR DESCRIPTION
**Pull request by genesisschaerrer**
I added Genesi Schaerrer as a contributor on the contribution_list.md

### CODE REVIEW

File: contribution_list.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 
 > Please add your name below to be added to the list of awesome people that have helped with this code.
 
+> Genesis Schaerrer
```
Comment (gleekzorp) [inline contribution_list.md]: Please update to be similar to the following.

```
- [Ciantera Rose](https://github.com/Ciantera-Rose)
- [Jacob Mason](github.com/JacobMason83)
```

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Since we have some others that used the link tags.  Lets follow that guideline.

File: contribution_list.md (hunk 1 above)
Comment (genesisschaerrer) [reply contribution_list.md]: got it, will change the layout to match the existing ones on the form

**Review by gleekzorp — APPROVED** [review]
Awesome!

**Pull request merged**